modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -117,3 +117,44 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetch autoload top list.
```

Comment:
Function/method descriptions should start with a third-person verb. :)
```suggestion
 * Fetches autoload top list.
```